### ISSUE TITLE
Ensure onboarding is navigatable using voice over

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#27709]
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-12722)

**Comment by data-sync-user** [comment]
➤ Razvan Litianu commented:

Hey Winnie Teichmann , I’m writing to get some visibility on this. I’m trying to figure out how voiceover should transition between elements. I think we need to go from top to bottom with a swipe inside cards, but I’m not sure when and how we can transition to the next card besides tapping buttons.

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

Razvan Litianu I think voice over should not advance to the next card by swiping to the next element after Not Now. The user should only get the next card by either double tapping the primary button or Not now or by swiping to the next card using 3 fingers. Also at the moment using the voice over swipe gesture the card is not put on screen correctly.

[^ScreenRecording_07-08-2025 10-14-30_1.MP4]

**Comment by data-sync-user** [comment]
➤ Razvan Litianu commented:

For QA: You should be able to navigate with voiceover by swiping between elements and swipe with 3 fingers between cards.

**Comment by data-sync-user** [comment]
➤ Miruna Curtean commented:

Validated on v142.1(60165), with iPhone 15(17.6) and on v142.1(60013), with iPad Mini (A17 Pro) with 18.0.

!Video Clip (2025-08-20 13_50_01).mp4|width=720,height=1560,alt="Video Clip (2025-08-20 13_50_01).mp4"!
